templates/common/_base/files/nmstate-configuration.yaml

```diff
@@ -5,25 +5,38 @@ contents:
     #!/bin/bash
     set -eux
 
+    # Clean up old config on behalf of mtu-migration
+    if ! systemctl -q is-enabled mtu-migration; then
+      echo "Cleaning up left over mtu migration configuration"
+      rm -rf /etc/cno/mtu-migration
+    fi
```

Comment:
I guess this makes the check in configure-ovs dead code?

Reply:
I suppose it does if we leave this service enabled everywhere.